README.md

```diff
@@ -1 +1,47 @@
-# InstaPy
+## InstaPy
+
+<img src="logo.png" width="200" align="right">
+
+#### A Collaborative Software Development Project
+Date: February 9, 2018
+
+Collaborators:
+
+Bhatnagar, Tarini : [Github Profile](https://github.com/tarinib)
+
+Guo, Xin (Alex): [Github Profile](https://github.com/alexguoxin)
+
+Nikel, Indiana: [Github Profile](https://github.com/indiana-nikel)
+
+#### Overview
+
+According to a [study](http://comp.social.gatech.edu/papers/icwsm15.why.bakhshi.pdf) by Yahoo Labs, “Filtered photos are 21 percent more likely to be viewed and 45 percent more likely to be commented on. Have you ever wondered how you could tranform your images using filters similar to Instagram in Python? 
+
+We present this package that performs digital image processingn using 2D convolutions. It uses a simple matrix as an image convolution kernel to do some interesting things! We have started with quite basic and diverse functionality and hope to advance and add more with time.
```

Comment:
Same as in InstaR, should we add a sentence about the greyscale and flip functions? As these two functions do not use 2D convolution